### ISSUE TITLE
Copy over all technical discovery sections to now follow the new format

### ISSUE THREAD

**Issue by abroddrick** [cisagov/manage.get.gov#3002]
### Issue description

In #3000 and #2998 we updated the technical discovery template and applied this to accounts and Zones. Now we just need to move everything thing else over into this format, so that each main section is it's own tab and follows the technical discovery template. This is a clean up task that should help in the readability of the document and allow the follow on technical discovery pieces to go more smoothly.

### Acceptance criteria

- [ ] Remove all technical discovery sections into their own tab and paste them into the format created in #2998 
- [ ] For any piece that has not had discovery start on it, if it has items written in paragraph form, covert the paragraph to bullet points where applicable

### Out of scope: 

- [ ]  You do not need to make each section more concise
- [ ] do not include CRUD operations or the Accounts &zones section

### Additional context

[technical discovery doc](https://docs.google.com/document/d/1Yq5d2M3MgM2vPhUBZ0k5wOmCQst4vND9-2qEZ55-h-Y/edit?pli=1&tab=t.0#heading=h.nl9i6mm1j4i3)

### Links to other issues

- [ ] blocked by #3000 
- [ ] blocked by #2998

**Comment by h-m-f-t** [comment]
Do we really anticipate this will be a 2?
